controller/attribute/util.py

```diff
@@ -0,0 +1,137 @@
+import docker
+import json
+import os
+import pytz
+import re
+from datetime import datetime
+from submodules.model.business_objects import attribute, record, project, tokenization
+from submodules.model.enums import DataTypes
+from submodules.s3 import controller as s3
+
+client = docker.from_env()
+image = os.getenv("AC_EXEC_ENV_IMAGE")
+exec_env_network = os.getenv("LF_NETWORK")
```

Comment:
not urgent in this PR, but i would recommend we change the name from `LF_NETWORK` to something more generic, as it now includes multiple envs.